src/settings/JobProfiles/JobProfiles.js

```diff
@@ -23,11 +29,14 @@ import {
 import { ViewJobProfile } from './ViewJobProfile';
 import { SettingPage } from '../SettingPage';
 import { resultsFormatter } from './resultsFormatter';
+import { LAYER_TYPES } from '../../utils/constants';
+import { createLayerURL } from '../../utils';
 
 import sharedCss from '../../shared.css';
 
-const INITIAL_RESULT_COUNT = 30;
-const RESULT_COUNT_INCREMENT = 30;
+// big numbers to get rid of infinite scroll
+const INITIAL_RESULT_COUNT = 100000;
```

Comment:
Let's specify `5000` for now. As we assume overall about 500 records I think 5000 is enough range above that.